Samples/PhotoEditor/README.md

```diff
@@ -0,0 +1,107 @@
+---
+page_type: sample
+languages:
+- cppwinrt
+- cpp
+products:
+- windows
+- windows-app-sdk
+name: "Photo Editor"
+urlFragment: PhotoEditor
+description: "Shows how to retrieve photos from the **Pictures** library, and then edit the selected image with assorted photo effects."
```

Comment:
```suggestion
description: "Shows how to retrieve photos from the Pictures library, and then edit the selected image with assorted photo effects."
```

Seems like the markdown bold format doesn't show up in the metadata format.

Reply:
Fixed